app/src/main/res/drawable/shape_circular_grey_textview.xml

```diff
@@ -2,9 +2,9 @@
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
   <item>
     <shape android:shape="oval">
-      <solid android:color="@color/grey_fog_light"/>
-      <stroke android:color="@color/grey_fog_light" android:width="1dp"/>
-      <size android:height="254dp" android:width="24dp"/>
+      <solid android:color="#6EBA15"/>
+      <stroke android:color="#336EBA15" android:width="6dp"/>
+      <size android:height="24dp" android:width="24dp"/>
```

Comment:
Why are we using colours not in the colors.xml - usually when there is this necessity we approximate the colour to the closes colors.xml one.